aodncore/pipeline/handlerbase.py

```diff
@@ -21,7 +21,7 @@
 from ..util import (ensure_regex_list, ensure_writeonceordereddict, format_exception,
                     get_file_checksum, iter_public_attributes, lazyproperty, matches_regexes, merge_dicts,
                     validate_relative_path_attr, TemporaryDirectory, WfsBroker, DEFAULT_WFS_VERSION)
-from ..version import __version__ as _aodncore_version
+from .._version import __version__ as _aodncore_version
```

Comment:
Would it be cleaner to just import it from the package root? (where we've already [handled the change](https://github.com/aodn/python-aodncore/blob/master/aodncore/__init__.py) in version source?)
```suggestion
from aodncore import __version__ as _aodncore_version
```